### PR TITLE
Add owood to releasers for support-core

### PR DESCRIPTION
**Pull request by Evildethow**
# Description

Adding owood to the existing releasers for https://github.com/jenkinsci/support-core-plugin. cc @batmat as existing maintainer for confirmation.

# Permissions pull request checklist

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ ] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by aheritier** [comment]
+1

File: permissions/plugin-support-core.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ developers:
 - "schristou"
 - "stephenconnolly"
 - "batmat"
-
+- "owood"
```
Comment (batmat) [inline permissions/plugin-support-core.yml]: I hope/suppose the parser isn't an old batshit and would fail without a newline...

Comment (daniel-beck) [inline permissions/plugin-support-core.yml]: Good news (sort of): When I created the initial batch of files, none had a trailing newline, which makes almost all diffs look like crap now.

**Comment by daniel-beck** [comment]
`owood` has never logged in (last item in the check list), so this isn't getting applied.

**Comment by aheritier** [comment]
@Evildethow ?? ^^^

**Comment by batmat** [comment]
Ping @christ66 FYI (very very sorry about that, my apologies, we should have pinged you here. I think many of us incorrectly assumed you would not want to be involved into this plugin anymore, at least in the short term :-)).

**Comment by Evildethow** [comment]
@daniel-beck @batmat @aheritier Sorry, missed the ping. Just logged into Artifactory to complete this.

**Pull request merged**